taxi/models.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+
+from django.db import models
+from django.db.models import CASCADE
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    def __str__(self):
+        return self.license_number
+
+
+class Manufacturer(models.Model):
```

Comment:
The `Meta` class is missing in the `Driver` model. According to the checklist, it's recommended to implement the `Meta` class after defining all attributes. You may not need it now, but it's a good practice to include it for future use.